libs/intl/src/useIntlContext.tsx

```diff
@@ -1,12 +1,12 @@
 import * as React from "react";
 
-import { IntlContext } from "./IntlProvider";
+import { IntlContext, IntlContextType } from "./IntlProvider";
 
 export function useIntlContext() {
   const context = React.useContext(IntlContext);
 
   if (!context) {
-    throw new Error("useIntlContext must be used within a IntlProvider");
+    return null as unknown as IntlContextType;
```

Comment:
this is hacky and will lead to problems. Add undefined as return value and check on usage if Context is defined